### ISSUE TITLE
jQuery Slim compatibility

### ISSUE THREAD

**Issue by fabiospampinato** [DataTables/DataTables#990]
Since jQuery v3 we also have a "[slim](https://blog.jquery.com/2016/06/09/jquery-3-0-final-released/)" version of it, which basically doesn't contain any code related to ajax or animation.

I think DataTables' core never uses `$.fn.animate` and only calls `$.ajax` in a handful of places, if those calls were made through an hypothetical `$.fn.dataTable.util.ajax` utility it would be possible to replace it with a custom implementation that uses [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API), or something else, instead.

The advantage would be 6 less kb of gzipped javascript served to the client by replacing jQuery with  jQuery Slim, nothing exceptional but it would be nice.

What do you think?

**Comment by DataTables** [comment]
Sounds like an excellent idea to me - thanks for the suggestion.

**Comment by abderrahmane-tj** [comment]
@DataTables I am thinking that this is not the right place but, Is there a way to tell DataTables how to do ajax ?

**Comment by DataTables** [comment]
Yes, the [forum](//datatables.net/forums) is where to ask support questions.

The [manual](https://datatables.net/manual/ajax) describes how you can get DataTables to Ajax load data. There are also [a number of examples](https://datatables.net/examples/ajax/index.html).
